include/matrix_operations/EigenvaluesProblems.h

```diff
@@ -439,10 +439,38 @@ class EigenvaluesProblems<NT, Eigen::Matrix<NT,Eigen::Dynamic,Eigen::Dynamic>, E
     /// \param[in] B: symmetric matrix
     /// \return The minimum positive eigenvalue and the corresponding eigenvector
     NT minPosLinearEigenvalue_EigenSymSolver(MT const & A, MT const & B, VT &eigvec) const {
+#if defined(SPECTRA_EIGENVALUES_SOLVER)
+	    int matrixDim = A.rows();
+    NT lambdaMinPositive;
+    
+    Spectra::DenseSymMatProd<NT> op(A);
+    Spectra::DenseCholesky<NT> Bop(B);
```

Comment:
The matrix A is the correlation matrix (the positive definite matrix). So, I guess the correct operations are:
    Spectra::DenseSymMatProd<NT> op(B);
    Spectra::DenseCholesky<NT> Bop(A);
    
We initially want to compute the smallest positive eigenvalue of the GEP, `A = tB`. So, here we need to compute the largest eigenvalue of the GEP, `tA = B` and invert it. That's why the operations has to be as above.